app/src/utils/paraspellTransfer.ts

```diff
@@ -149,14 +150,15 @@ export const getTransferableAmount = async (
 
   const currencyId = getParaspellToken(sourceToken, sourceChainNode)
 
-  return await Builder(wssEndpoint)
+  const transferableAmount = await Builder(wssEndpoint)
     .from(sourceChainNode as TNodeDotKsmWithRelayChains)
     .to(destinationChainNode)
-    // Pass a dummy amount
-    .currency({ ...currencyId, amount: 1000n })
+    .currency({ ...currencyId, amount: userBalance })
     .address(recipient)
     .senderAddress(sender)
     .getTransferableAmount()
+
+  return (transferableAmount * 99n) / 100n // 1% Temporary fix as PS investigate getTransferableAmount issue
```

Comment:
```suggestion
  return (transferableAmount * 0.99) // Slice 1% off as a temporary fix as PS investigate getTransferableAmount issue
```
can't we simplify here?